lua/gatekeeper.lua

```diff
@@ -99,6 +99,11 @@ local ffi = require("ffi")
 -- Structs
 ffi.cdef[[
 
+struct constants {
+	static const uint32_t BONDING_MODE_ROUND_ROBIN = 0;
+	static const uint32_t BONDING_MODE_8023AD = 4;
```

Comment:
1. Why didn't you include the other modes?

2. Check the examples in `lua/policylib.lua` on how to bring C constants into Lua. It's cleaner than the way you're doing now.